draft-ietf-gnap-core-protocol.md

```diff
@@ -3525,6 +3525,10 @@ by both clients and ASs, which could be accomplished by e.g. a client publishing
 a public key at a URL. For interoperability, this method could later be described
 as an extension.
 
+### Key Protection {#key-protection}
+
+The security of GNAP when in use with symmetric keys relies on the cryptographic security of the keys themselves. 
+Therefore, a key management system MUST be used to supply the keys. Moreover, the symmetric key MUST NOT be a human memorable password or a value derived from one.
```

Comment:
I like what you're doing here, but I think maybe we can do something like this instead:

```suggestion
The security of GNAP relies on the cryptographic security of the keys themselves. 
When used with symmetric keys, a key management system or secure key derivation mechanism MUST be used to supply the keys. Symmetric keys MUST NOT be a human memorable password or a value derived from one.
```